src/Standards/PEAR/Sniffs/WhiteSpace/ScopeClosingBraceSniff.php

```diff
@@ -92,6 +92,18 @@ public function process(File $phpcsFile, $stackPtr)
             }
         }
 
+        // Allow empty braces with multiline parenthesis.
+        if ($tokens[$stackPtr]['type'] === 'T_FUNCTION') {
+            $parenthesisStart = $tokens[$stackPtr]['parenthesis_opener'];
+            $parenthesisEnd   = $tokens[$stackPtr]['parenthesis_closer'];
+            if ($tokens[$parenthesisStart]['line'] !== $tokens[$parenthesisEnd]['line']
```

Comment:
Perhaps we can allow empty body for all methods and functions without checking of parenthesis lines diff:

```php
class ConstructorPropertyPromotion
{
    public function __construct(protected int $id) {}
}
```